low-level/files/opt/seagate/sspl/prerequisites/sspl-prereqs.sh

```diff
@@ -0,0 +1,320 @@
+elif [ -n "$TARGET_BUILD" ]; then
+    ./setup_repos.sh -t $TARGET_BUILD 2>&1 | tee -a ${LOG_FILE}
+else
+    ./setup_repos.sh 2>&1 | tee -a ${LOG_FILE}
+fi
+rm -rf setup_repos.sh
+echo -e "\nSetup repos - Done" 2>&1 | tee -a ${LOG_FILE}
+
+
+# Install prereq script dependencies
+yum install -y python3
+pip3 install configobj
+
+# Install required RPMS if not available
+"$install_3rd_party_packages" && {
+    rpm -qa | grep "cortx-py-utils" || {
+        yum install -y cortx-py-utils 2>&1 | tee -a ${LOG_FILE}
+    }
+    rpm -qa | grep "consul" || {
+        yum install -y consul 2>&1 | tee -a ${LOG_FILE}
+    }
+    rpm -qa | grep "rabbitmq-server" || {
+        yum install -y rabbitmq-server 2>&1 | tee -a ${LOG_FILE}
+    }
+}
+
+
+setup_consul(){
+    if ! [ -x "$(command -v consul)" ]; then
+        echo "Consul is not available. \
+        For consul and other 3rd party package installation, \
+        check prereq script usage. Exiting."
+        exit 1
+    fi
+    # Create config file for consul agent
+    CONSUL_CONFIG_DIR='/opt/seagate/cortx/sspl/bin'
+    CONSUL_CONFIG_FILE="$CONSUL_CONFIG_DIR/consul_config.json"
+    if [[ ! -e "$CONSUL_CONFIG_DIR" ]]; then
+        mkdir -p $(dirname "${CONSUL_CONFIG_DIR}")
+    fi
+    cat << EOF >> $CONSUL_CONFIG_FILE
+        {
+            "watches": [
+                {
+                    "type": "key",
+                    "key": "sspl/config/SYSTEM_INFORMATION/log_level",
+                    "args": ["/opt/seagate/cortx/sspl/bin/consume_cfg_change_alert"]
+                }
+            ]
+        }
+EOF
+    # Invoke consul agent from fallback path
+    CONSUL_PS=$(pgrep "consul" || true)
+    [ -z "$CONSUL_PS" ] &&
+        consul agent -dev -config-file=$CONSUL_CONFIG_FILE &>/dev/null &
+
+    TRIES=0
+    while [ -z "$CONSUL_PS" ]; do
+        sleep 2
+        TRIES=$((TRIES+1))
+        CONSUL_PS=$(pgrep "consul" || true)
+        if [ $TRIES -gt 5 ]; then
+            echo "Consul service is not started"
+            break
+        fi
+    done
+
+    # Choose sspl.conf file based on component version
+    SSPL_CONF='/etc/sspl.conf'
+    curl $CORTX_MONITOR_BASE_URL/low-level/files/opt/seagate/sspl/conf/sspl.conf.${SSPL_VERSION} -o sspl.conf;
+    mv sspl.conf $SSPL_CONF
+
+    # Load sspl conf to consul
+    curl $CORTX_MONITOR_BASE_URL/low-level/files/opt/seagate/sspl/prerequisites/common_config_SN.ini -o common_config_SN.ini;
+    curl $CORTX_MONITOR_BASE_URL/low-level/files/opt/seagate/sspl/prerequisites/feed_sspl_conf_to_consul.py -o feed_sspl_conf_to_consul.py;
+    chmod a+x feed_sspl_conf_to_consul.py
+
+    echo "Inserting common config in consul.."
+    python3 feed_sspl_conf_to_consul.py -F common_config_SN.ini -N $NODE -A $CNTRLR_A \
+                    -B $CNTRLR_B -U $CNTRLR_USER -P $CNTRLR_PASS -Ap $CNTRLR_A_PORT \
+                    -Bp $CNTRLR_B_PORT -bi $BMC_IP -bu $BMC_USER -bp $BMC_PASS
+    rm -rf common_config_SN.ini
```

Comment:
why removing, let it be available for debugging in pre-reqs folder. same for line 295

Reply:
Okay